dd-sdk-android/src/main/kotlin/com/datadog/android/Datadog.kt

```diff
@@ -24,6 +24,8 @@ import com.datadog.android.privacy.TrackingConsent
 import com.datadog.android.rum.internal.RumFeature
 import com.datadog.android.tracing.internal.TracesFeature
 import java.lang.IllegalArgumentException
+import java.util.ArrayList
```

Comment:
redundant imports

Reply:
Yes...will clean and add the tests after we decide which way to go